.travis.yml

```diff
@@ -14,19 +12,29 @@ test_stage: &test_stage
 jobs:
   include:
     - stage: lint
+      before_install:
+        - npm install -g npm@latest
       node_js: 8
       script:
         - npm run lint
     - stage: test
+      before_install:
```

Comment:
IIRC, `before_install` can be included in the reference, then overridden for this specific version.